sdk/storage/azure-storage-blob-changefeed/azure/storage/blob/changefeed/_change_feed_client.py

```diff
@@ -24,9 +24,11 @@ class ChangeFeedClient(object):  # pylint: disable=too-many-public-methods
         The URI to the storage account.
     :param credential:
         The credentials with which to authenticate. This is optional if the
-        account URL already has a SAS token. The value can be a SAS token string, an account
+        account URL already has a SAS token. The value can be a SAS token string,
+        an instance of a AzureSasCredential from azure.core.credentials, an account
         shared access key, or an instance of a TokenCredentials class from azure.identity.
-        If the URL already has a SAS token, specifying an explicit credential will take priority.
+        If AzureSasCredential is used, the URI must not contain a SAS token otherwise
+        if the URL already has a SAS token, specifying an explicit credential will take priority.
```

Comment:
Shouldn't this say "a ValueError error will be raised" rather than "...will take priority"?